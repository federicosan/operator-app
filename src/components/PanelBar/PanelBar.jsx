import React, { Component, PropTypes } from 'react';
import { connect } from 'react-redux';

export class PanelBarComponent extends Component {
  static propTypes = {
    dispatch: PropTypes.func,
  }

  constructor (props) {
    super(props);
  }

  render () {
    return (
      <div className="row" />
    );
  }
}

const mapStateToProps = state => ({

});

const mapDispatchToProps = dispatch => ({
  dispatch,
});

const PanelBar = connect(
  mapStateToProps,
  mapDispatchToProps,
)(PanelBarComponent);

export default PanelBar;
