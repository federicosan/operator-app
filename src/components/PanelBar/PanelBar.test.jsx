import React from 'react';

import { shallow } from 'enzyme';

import PanelBar from './PanelBar.jsx';

const store = {
  subscribe: jest.fn(),
  dispatch: jest.fn(),
  getState: jest.fn(() => ({ })),
};

describe('PanelBar', () => {
  it('matches snapshot', () => {
    const component = shallow(<PanelBar store={store} />);

    expect(component).toMatchSnapshot();
  });
});
