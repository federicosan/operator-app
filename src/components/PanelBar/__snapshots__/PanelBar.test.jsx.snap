exports[`PanelBar matches snapshot 1`] = `
<PanelBarComponent
  dispatch={[Function]}
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "subscribe": [Function],
    }
  } />
`;
